mitosheet/src/mito/components/endo/CellDropdown.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Saga Inc.
+
+import React from 'react';
+import { ActionEnum, UIState } from '../../types';
+import Dropdown from '../elements/Dropdown';
+import DropdownItem from '../elements/DropdownItem';
+import DropdownSectionSeperator from '../elements/DropdownSectionSeperator';
+import { TaskpaneType } from '../taskpanes/taskpanes';
+import { getPropsForContextMenuDropdownItem } from './utils';
+import { Actions } from '../../utils/actions';
+import { isCurrOpenDropdownForCell } from './visibilityUtils';
+
+/*
+    Displays a set of actions one can perform on a row
+*/
+export default function CellDropdown(props: {
```

Comment:
Now might be a nice time to rename all of these to `ContextMenu` -- all three of em